configure.ac:  override CONFIGURE_LINE

For reproducible binaries, we need to report a constant CONFIGURE_LINE.
This patched is only applied for such builds.

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Joe Slater <joe.slater@windriver.com>


--- git.orig/configure.ac
+++ git/configure.ac
@@ -15,7 +15,10 @@ AC_PREREQ(2.69)
 ################################################################################
 dnl -- Process this file with autoconf to produce a configure script.
 AC_INIT
-CONFIGURE_LINE="$0 $@"
+dnl The default CONFIGURE_LINE has been overridden for reproducible binaries
+dnl CONFIGURE_LINE="$0 $@"
+CONFIGURE_LINE="configure parameters are not available for reproducible builds"
+
 AC_CONFIG_SRCDIR([lib/device/dev-cache.h])
 AC_CONFIG_HEADERS([include/configure.h])
 
